actions.yaml

```
@@ -119,6 +115,10 @@ migrate-node-key:
       type: boolean
       description: Undo the migration by moving the node key back to the legacy location
       default: false
+    snap-name:
+      type: string
+      description: Name of the snap
+      enum: [ polkadot, 'polkadot-parachain' ]
```

Nit: quotes around `polkadot-parachain` should not be necessary

```suggestion
      enum: [ polkadot, polkadot-parachain ]
```